src/listconf.c

```diff
@@ -323,9 +323,7 @@ static void listconf_list_build_info(void)
 	printf("times(2) CLK_TCK is %ld\n", clk_tck);
 #endif
 #if defined (__MINGW32__) || defined (_MSC_VER)
-	printf("Using clock(3) for timers, claimed resolution %ss, observed %ss\n",
-	       human_prefix_small(1.0 / CLOCKS_PER_SEC),
-	       human_prefix_small(1.0 / sm_cPrecision));
+	printf("Using clock(3) for timers, claimed resolution %ss\n", human_prefix_small(1.0 / CLOCKS_PER_SEC));
```

Comment:
You wrote "Also drops obsolete fallbacks to clock(3)", so I expected the code would no longer use `clock(3)` at all, but here you still have a message claim that it does?

Reply:
No, that line refers to the original timers in `signals.c`